app/views/posts/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1>Here is the detail for a given post</h1>
+<div class="user-container">
+    <div class= 'post-container'>
+            <div class="post-wrapper">
+                    <h2>Post #</h2>
+                    <p><%= @post.text %></p>
+                    <h5>Comments: <%= @post.comments_counter %> , Likes: <%= @post.likes_counter %></h5>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Aspernatur voluptates dicta similique ab soluta totam inventore ratione, perferendis facilis facere maiores. Modi impedit aperiam dolores tempora eligendi odit et repellendus laboriosam nihil, deleniti aut facere vel doloribus possimus rem quia aliquam nesciunt? Dolorum veritatis molestiae vero quod molestias exercitationem aut.</p>
+            </div>
+            <div class="comment-wrapper">
+                <% @post.recent_comments.each_with_index do |comment,index| %>
+                    <h4><%= comment.author.name %> : <%= "#{comment.text}: # #{index+1}" %></h4>
+                <% end %>
```

Comment:
- [ ] I noticed that you are still rendering the 5 most recent comments on the post show page. Please show all of the comments for a specific post 👍 
![image](https://github.com/addisalemseifu/my_blog_app/assets/46839548/f06d5336-6e42-4698-ac82-e11b5db48f20)

Reply:
Well resolved ✔️ 

![image](https://github.com/addisalemseifu/my_blog_app/assets/38088818/868e4eb0-fede-4d07-80e8-336d68f78347)